src/components/App.js

```diff
@@ -23,38 +25,35 @@ export default class App extends Component {
   }
 
   typeHandler(e, stateType){
```

Comment:
on line 27 you declare your function without a space after the parentheses, - (){

on line 38 you declare with a space - () {

super small detail, overall it looks pretty good